EIPS/eip-2502.md

```diff
@@ -0,0 +1,462 @@
+---
+eip: 2502
+title: µTopic Delegation
+author: Ricardo Guilherme Schmidt (@3esmit)
+discussions-to: https://ethereum-magicians.org/t/eip-2502-topic-delegation/3974
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-02-02
+requires: 2470
+
+---
+
+## Simple Summary
+
+A common trust network for ethereum.
+
+A general purpose hierarchy of delegations which can divided into micro topics. 
+
+## Abstract
+
+When "everything" becomes DAOable, it could become a problem to keep track of all the micro DAOs. Just like ERC20 have an allowance for other address spent on their behalf, 
+Cold wallets usually don't want to be unlocked very often to be able to vote. 
+Users might want to trust all their DAOs to other address for vote for them. 
+A group of people want to trust each other to vote in topics.
+DAOs might want to inherit the _root_ delegate of users for their own _topic_, where users could define a specific delegate..
```

Comment:
```suggestion
```
This all sounds like great content for the `Motivation` section, but not really appropriate for the `Abstract` section.  The abstract should be a human readable form of the specification.  Someone should be able to read the abstract and get the gist of the specification itself.  Someone should be able to read the motivation section and get an idea of why someone may want to implement this specification.